bincode-transport/tests/pushback.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the MIT License, <LICENSE or http://opensource.org/licenses/MIT>.
+// This file may not be copied, modified, or distributed except according to those terms.
+
+//! Tests client/server control flow.
+
+#![feature(generators, await_macro, async_await, futures_api,)]
+
+#[macro_use]
+extern crate log;
+#[macro_use]
```

Comment:
It's bad style to use macro_use now that you use 2018 edition. I'd suggest importing which macros you need specifically

Reply:
Done.